comms/Arduino-CC1120x/main.cpp

```diff
@@ -0,0 +1,197 @@
+#include "Arduino-CC1120x.h"
```

Comment:
The main file should be the .ino file, and the library or source file should be the .cpp file. Sorry, should have provided more instructions on that.

Reply:
The library or source file should also have #include "Arduino.h" statement.